### PR TITLE
2020.11 docs

### PR DESCRIPTION
**Pull request by antgonza**
First review/merge https://github.com/biocore/qiita/pull/3048

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,13 +39,7 @@ Current features
...
   part of the European Bioinformatics Institute (EBI) for private and public
   studies.
-* Raw data processing for:
-
-  * Target gene data: we support deblur against GreenGenes (13_8) and close
-    reference picking against GreenGenes (13_8) and Silva.
-  * Metagenomic and Metatranscriptomic data: we support Shogun processing.
-  * biom files can be added as new preparation templates for downstream
-    analyses; however, this cannot be made public.
+* Raw data processing for `Target Gene, Metagenomic, Metabolomic and BIOM files <https://qiita.ucsd.edu/static/doc/html/processingdata/index.html#processing-recommendations>`. BIOM files can be added as new preparation templates for downstream analyses; however, this cannot be made public.
```
Comment (ElDeveloper) [inline README.rst:42]:
```suggestion
* Raw data processing for `Target Gene, Metagenomic, Metabolomic and BIOM files <https://qiita.ucsd.edu/static/doc/html/processingdata/index.html#processing-recommendations>`. BIOM files can be added as new preparation files for downstream analyses; however, this cannot be made public.
```

File: qiita_pet/support_files/doc/source/faq.rst [hunk 2]
```diff
@@ -30,8 +30,8 @@ What kind of data can I upload to Qiita for processing?
 Processing in Qiita requires 3 things: raw data, sample and prep information
 files. `Here <https://github.com/biocore/qiita/blob/master/README.rst#accepted-raw-files>`__
 you can find a list of currently supported raw files files. Note that we are
-accepting any kind of target gene (16S, 18S, ITS, whatever). You can also upload
-and process WGS via Shogun. Check our :doc:`processingdata/processing-recommendations`.
+accepting any kind of target gene (16S, 18S, ITS, whatever), Whole Genome and
+Metrascriptomic. Check our :doc:`processingdata/processing-recommendations`.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/faq.rst:34]:
```suggestion
Metatranscriptomic. Check our :doc:`processingdata/processing-recommendations`.
```

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 3]
```diff
@@ -8,8 +8,10 @@ Currently, Qiita supports the processing of raw data from:
 #. Metatranscriptome sequencing
 
 
-Note that the selected processing are mainly guided so we can perform meta-analyses, this is combine different studies,
-even from different wet lab techniques or sequencing technologies.
+Note that the selected processing are mainly guided so we can perform meta-analyses,
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:11]:
```suggestion
Note that the selected processing recommendations are mainly guided towards performing meta-analyses,
```

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 4]
```diff
@@ -41,13 +43,15 @@ Currently, we have the reference databases: Greengenes version 3_8-97, Silva 119
 Shotgun sequencing
 ------------------
 
-Qiita currently has one shotgun metagenomics data analysis pipeline: `Shogun <https://msystems.asm.org/content/3/6/e00069-18>`_.
+Qiita currently has one active shotgun metagenomics data analysis pipeline: a per sample
+bowtie2 alignment step with Woltka classification using either the WoLr1 or Rep200 databases.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:47]: What's `WoLr1`?

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
Just a few minor suggestions/questions. Looks good overall.

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 4 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:47]: WoLr1 ("Web of Life" release 1)

**Review by ElDeveloper — APPROVED** [review]
Thanks @antgonza

**Pull request merged**